src/js/components/tablero/TableCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* global _ */
+const TableCard = props => (
+  <div className="tableCard">
+    <table>
+      <thead>
+        <tr>
+          <th>{props.data.head.number}</th>
+          <th>{_.truncate(props.data.head.name, { length: 50, omission: '...' })}</th>
+          <th>{_.truncate(props.data.head.value, { length: 50, omission: '...' })}</th>
+        </tr>
+      </thead>
+      {props.data.body.map(item => (
+        <tbody key={`item-${Math.random()}`} className="tableLink">
```

Comment:
Perhaps you could use item.id here instead of random (if you have access to item.id in here)